hedwiki/score/HED_score_0.0.2.mediawiki

```diff
@@ -675,7 +675,7 @@ This project was supported by the by the National Institute of Mental Health of
                 **** Typical-absence-epileptic-seizure
                 **** Atypical-absence-epileptic-seizure
                 **** Myoclonic-absence-epileptic-seizure
-                **** Eyelid-myoclonia-epileptic-seizure
+                **** Eyelid-myoclonia-absence-epileptic-seizure
```

Comment:
Should `absence` be `absent` ?

Reply:
should remain  absence - it's an absence seizure (aka petit mal)